src/basset.cpp

```diff
@@ -205,14 +245,14 @@ void CompilationDatabase::add(const string &directory,
 
       *this << "\n"
                // clang-format off
-               "      \"" << arg << '\"';
-      // clang-format on
+               "      \"" << json_escape(arg) << '\"';
+               // clang-format on
```

Comment:
As you can guess, I am using `clang-format` to format the code. Actually, I am using _Format Document_ command in VS Code, which relies on `clangd`, which uses `clang`'s formatting library... But the result should be the same as with `clang-format`.

The version of `clangd` I am using (it is the installed in Gitpod environment):
```
$ clangd --version
Ubuntu clangd version 17.0.6 (++20231209124227+6009708b4367-1~exp1~20231209124336.77)
Features: linux+grpc
Platform: x86_64-pc-linux-gnu
```

This version of `clangd`/`clang-format` for some reason puts a funny indent for this line. It is a bit annoying, but adjusting the indent after every _Format Document_ would be even worse. So, could you please restore the original indentation level?